blueprints/devops/goreleaser/ops2deb.yml

```diff
@@ -0,0 +1,18 @@
+name: goreleaser
+matrix:
+  architectures:
+    - amd64
+  versions:
+    - 2.0.1
+homepage: https://goreleaser.com
+summary: release Go projects as fast and easily as possible
+description: |-
+  GoReleaser is an open-source tool that simplifies the process of releasing your
+  Go projects to multiple platforms, ensuring a consistent and reliable
+  deployment experience.
+fetch:
+  url: https://github.com/goreleaser/goreleaser/releases/download/v{{version}}/goreleaser_Linux_{{target}}.tar.gz
+  targets:
+    amd64: x86_64
+install:
+  - goreleaser:{{src}}/usr/bin/
```

Comment:
I'm not entirely sure what the result of `{{src}}` would be after it's _resolved_, but I copied it as is from other manifests.

Reply:
You don't need src here, just:

```
install:
  - goreleaser:/usr/bin/
```